trunk/src/modules/mpi_wrapper.F90

```diff
@@ -58,6 +59,9 @@ subroutine mpi_w_init()
 !
 !  ...set initialization flag
       MPI_IS_INIT = .true.
+!
+!      call MPI_Errhandler_set(MPI_COMM_WORLD,MPI_ERRORS_RETURN,ierr);
+!      call mpi_w_handle_err(ierr,'MPI_ERR_HANDLER')
```

Comment:
is this needed for the change below (calling `mpi_error_string`)?

Reply:
I  don't entirely remember; I don't think so since it's commented out though

Reply:
I remember now, this causes MPI to return on error; it helps with MPI debugging because sometimes the first error code it returns wants you to check an error elsewhere. If enabled this will cause the code to continue even after MPI reports an error, which can then lead to segfaults later (and complicate finding bugs) so it should only be enabled when needed for debugging.